Source/AliveLibAO/Input.cpp

```diff
@@ -312,7 +312,7 @@ EXPORT void InputObject::Update_433250()
         {
             const u32 command = (*field_18_demo_res)[field_1C_demo_command_index++];
             field_24_command = command >> 16;
-            field_28_command_duration = gnFrameCount_507670 + command & 0xFFFF;
+            field_28_command_duration = (gnFrameCount_507670 + command) & 0xFFFF;
```

Comment:
same bug we saw in AE, the &0xFFFF is supposed to be on `command` as its being split into hi and lo

Reply:
Will fix